### ISSUE TITLE
Error with GLM in MLencoding notebook

### ISSUE THREAD

**Issue by lachioma** [KordingLab/spykesML#5]
There is some error with the implementation of pyglmnet. See attached screenshot.

![image](https://user-images.githubusercontent.com/29898879/81455789-2f9d1b00-915e-11ea-85c1-af82d6d464be.png)

**Comment by aribenjamin** [comment]
Hi, thanks for the heads up! There were some recent changes to the pyglmnet API and it looks like this broke some things. I'll change this soon, within a few weeks, and keep this open until then. (if you need it fast it's probably best to jump right to pyglmnet for now)

**Comment by titipata** [comment]
@aribenjamin, yeah, we no longer accept `reg_lambda` in the range [here](https://github.com/KordingLab/spykesML/blob/master/MLencoding/mlencoding.py#L159). It now looks something like [this](https://github.com/glm-tools/pyglmnet/blob/master/examples/plot_glm_retinal_ganglia.py#L161-L164). Basically, you just need to create multiple GLM instances for each `reg_lambda` and that should solve this problem! (Also, you might have to edit [here](https://github.com/KordingLab/spykesML/blob/master/MLencoding/mlencoding.py#L241-L242))

**Comment by lachioma** [comment]
thank you!
